_pylibczi/Image.h

```diff
@@ -23,7 +26,7 @@ namespace pylibczi {
    * one subblock which may be either 2D or 3D, in the case of 3D the data is then later split into multiple 2D Image<T> so that
    * the concept of a Channel isn't destroyed.
    */
-  class Image {
+  class Image : public SubblockSorter {
```

Comment:
this inheritance feels a bit odd to me. What is a SubblockSorter?  could Image hold onto one as a member instead of deriving from it?  Where do Images get treated polymorphically as SubblockSorters?

Reply:
After discussing, I think SubblockSorter could be renamed to SubblockSortable.  The key thing to know is that terminology-wise, Subblock and and Image have a one to one relationship.  A sub-block can be thought of as an image.